doc/source/user/basics.dispatch.rst

```diff
@@ -56,7 +56,7 @@ array([[2., 0., 0., 0., 0.],
 
 Notice that the return type is a standard ``numpy.ndarray``.
 
->>> type(arr)
+>>> type(np.multiply(arr, 2))
```

Comment:
@rossbar another fix would be

```suggestion
>>> type(arr_times_2)
```

and also change line 49 to be `arr_times_2 = np.multiply(arr, 2)`